caracal/schema/prep_schema.yml

```diff
@@ -28,7 +28,27 @@ mapping:
             desc: Enable the 'fixvis' segment.
             type: bool
             required: false
-            example: 'False'            
+            example: 'False'         
+      fixcalcoords:
+        desc: Fix the coordinates of the bandpass calibrators.
+        type: map
+        mapping: 
+          enable: 
+            desc: Enable fixing calibrator coodinate fixing segment.
+            type: bool
+            required: false
+            example: 'False'
+          tol: 
+            desc: Tolerance (in radians) for matching calibrator to database names.
```

Comment:
Would arcsec not be a better unit here? I find it very hard to think of small offsets in radians...

Reply:
Pure laziness on my part - let me fix it.

Reply:
Done.